TASVideos.Core/Services/ForumService.cs

```diff
@@ -257,41 +284,72 @@ public async Task<int> CreatePost(PostCreateDto post)
 				tkey => tkey.Key,
 				tvalue => tvalue
 					.GroupBy(ffr => ffr.TopicId)
-					.ToDictionary(ttkey => ttkey.Key, ttvalue => ttvalue.Max(fffr => fffr.CreateTimestamp)));
-
-		_cacheService.Set(TopicActivityCacheKey, forumActivity, Durations.OneDayInSeconds);
+					.ToDictionary(
+						ttkey => ttkey.Key,
+						ttvalue => (
+							JsonSerializer.Serialize(ttvalue
+								.Where(fffr => fffr.CreateTimestamp > minimumDate)
+								.ToDictionary(
+									tttkey => tttkey.Id,
+									tttvalue => tttvalue.CreateTimestamp.UnixTimestamp())),
+							JsonSerializer.Serialize(ttvalue
+								.Where(fffr => fffr.PostEditedTimestamp != null && fffr.PostEditedTimestamp > minimumDate)
+								.ToDictionary(
+									tttkey => tttkey.Id,
+									tttvalue => ((DateTime)tttvalue.PostEditedTimestamp).UnixTimestamp()))
+						)));
```

Comment:
Turning the `tvalue =>` and `ttvalue =>` lambdas into local methods would make this more readable.